src/scss/components/_header.scss

```diff
@@ -10,40 +10,205 @@
     justify-content: space-between;
     margin: 0 auto;
     max-width: 1440px;
-    padding: 2.4rem 4.8rem;
+    padding: 2rem 2.4rem;
     width: 100%;
   }
 
   &__logo {
-    width: 135px;
+    width: 112px;
 
     &__image {
       width: 100%;
     }
   }
 
   &__menu {
-    align-items: center;
+    background-color: var(--white);
     display: flex;
-    gap: 4rem;
+    flex-direction: column;
+    gap: 0;
+    justify-content: space-between;
```

Comment:
I think you can remove this. The space between items is defined by the margins.

Reply:
I tried removing the gag too and there is no problem, thanks